mmv1/templates/terraform/iam/example_config_body/vertex_ai_featurestore.tf.erb

```diff
@@ -0,0 +1,3 @@
+  project = google_vertex_ai_featurestore.featurestore.project
+  region = google_vertex_ai_featurestore.featurestore.region
+  featurestore = google_vertex_ai_featurestore.featurestore.name
```

Comment:
[MEMO] Without this custom `example_contig_body`, the region was defined twice in the IAM resources. That'd be because `base_url` needs `{{region}}` in addition to the parent resource. 

https://github.com/GoogleCloudPlatform/magic-modules/blob/aa5cc7500ee772636fb2a68d345623dd00076102/mmv1/products/vertexai/api.yaml#L23